### ISSUE TITLE
Cannot Install on Fedora 25

### ISSUE THREAD

**Issue by myc132** [kryptco/kr#179]
Hello,

I had an older version of kr on my machine and I tried to upgrade using `kr upgrade` and found that there was an issue so I uninstalled it using `dnf remove kr` and went to the krypt.co and followed the instructions to install.

It did not complete and now when i try to do `sudo dnf install kr` i see the following

```sudo dnf install kr
created by dnf config-manager from https://kryptco.github.io/yum                                                                                                                                              30 kB/s | 4.5 kB     00:00    
Dependencies resolved.
=============================================================================================================================================================================================================================================
 Package                                        Arch                                               Version                                                           Repository                                                         Size
=============================================================================================================================================================================================================================================
Installing:
 kr                                             x86_64                                             2.3.1-2.el7.centos                                                kryptco.github.io_yum                                              11 M

Transaction Summary
=============================================================================================================================================================================================================================================
Install  1 Package

Total download size: 11 M
Installed size: 43 M
Is this ok [y/N]: y
Downloading Packages:
[MIRROR] kr-2.3.1-2.el7.centos.x86_64.rpm: Interrupted by header callback: Server reports Content-Length: 12048980 but expected size is: 12047868                                                                                           
[FAILED] kr-2.3.1-2.el7.centos.x86_64.rpm: No more mirrors to try - All mirrors were already tried without success                                                                                                                          

The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: Error downloading packages:
  Cannot download kr-2.3.1-2.el7.centos.x86_64.rpm: All mirrors were tried
```

I tried to clean the cache and it still will not work.

**Comment by kcking** [comment]
Thank you for reporting, there was an issue with the 2.3.1-2 release, which should now be corrected with 2.3.1-3.

Please try again and let us know if the new release works for you.

**Comment by myc132** [comment]
Thanks works now!
